update-ngxblocker

```diff
@@ -56,6 +61,10 @@ Usage: $script [OPTIONS]
         [ -i ] : Change installer path         (default: $INSTALLER)
         [ -r ] : Change repo url               (default: $REPO)
         [ -e ] : Change @email address         (default: $EMAIL)
+        [ -g ] : Change @email address Mailgun (default: $EMAIL)
+        [ -d ] : Set Mailgun Domain
+        [ -a ] : Mailgun API Key
+        [ -f ] : From Address
```

Comment:
` [ -f ] : Mailgun From Address`

so users do not think it will change the system `FROM` address

Reply:
done